services-js/registry-certs/client/birth/components/VerifyIdentificationComponent.tsx

```diff
@@ -107,7 +112,10 @@ export default class VerifyIdentificationComponent extends React.Component<
         </ul>
 
         <div className="g">
-          <div className="g--6 m-v500">
+          <div
+            className="g--6 m-v500"
+            onClick={() => this.handleSendEventClick('upload front of ID')}
```

Comment:
just double-checking that these are all running (that the event’s bubbling wasn't stopped).

If so, 👍